fed/mosaic-sns/src/main/java/org/eclipse/mosaic/fed/sns/ambassador/SnsAmbassador.java

```diff
@@ -185,13 +186,17 @@ private void process(AdHocCommunicationConfiguration interaction) {
                 if (configuration.getConf0() != null && configuration.getConf0().getRadius() != null) {
                     communicationRadius = configuration.getConf0().getRadius();
                 } else {
-                    log.warn("Node {} is not configured with a distance value. Using global singlehop radius from SNS configuration.", nodeId);
+                    log.debug("Node {} is not configured with a distance value. Using global singlehop radius from SNS configuration.", nodeId);
                 }
                 if (SimulationEntities.INSTANCE.isNodeSimulated(nodeId)) {
                     SimulationEntities.INSTANCE.enableWifi(nodeId, communicationRadius);
                 } else {
+                    // Vehicle has not been simulated by the SNS, yet. Radio is configured for the first time.
                     registeredVehicles.put(nodeId, communicationRadius);
+                    log.info("Added Vehicle id={} @time={}", nodeId, TIME.format(interaction.getTime()));
```

Comment:
I'm not sure if this is correct. I would print this message when the vehicle goes "online" and not when it is "pre-registered". So moving this line to 192 would be correct, wouldn't it?

Reply:
I thought this was intended. Maybe we should discuss it with @rprotzmann .

Reply:
Change DEBUG and write "Pre-register radio configuration for vehicle id=..."